nimbus-core/src/main/java/com/antheminc/oss/nimbus/domain/defn/ViewConfig.java

```diff
@@ -1682,6 +1682,11 @@ public static boolean isValidNumericFilter(FilterMode mode) {
 		String cssClass() default "dropdownTrigger";
 
 		String imgSrc() default "";
+		
+		/**
+		 * <p>When set to true the link menu will collapse on clickin a link
```

Comment:
1. You can do code styling with `{@code someCodeValue}` to illustrate a code value. Consider giving `{@code true}`.

2. Typo. Please change `"clickin"` to `"clicking"`